c21e/ruby/default.mk

```diff
@@ -31,12 +31,12 @@ endif
 
 publish: .deps
 	gem build $(GEMSPEC)
-	gem push $$(find . -name "*.gem")
+	gem push $$(find . -name "*$(NEW_VERSION).gem")
 .PHONY: publish
 
 clean: clean-ruby
 .PHONY: clean
 
 clean-ruby:
-	rm -f .deps .linked .tested Gemfile.lock
+	rm -f .deps .linked .tested Gemfile.lock .gem
```

Comment:
Did you mean `*.gem`?

Reply:
If so, should this be done in the `.template/ruby/default.mk` file instead?

Reply:
Never mind the last comment, I see you added it to `.template/ruby/default.mk`, but I still think it should be `*.gem`